app/models/user.rb

```diff
@@ -22,6 +24,24 @@ def digest string
       end
       BCrypt::Password.create(string, cost: cost)
     end
+
+    def new_token
+      SecureRandom.urlsafe_base64
+    end
+  end
+
+  def remember
+    @remember_token = User.new_token
+    update remember_digest: User.digest(@remember_token)
```

Comment:
à lúc đầu em viết update_attribute thiếu chữ "s" ở cuối rubocop nó báo method not validate :v sau em tìm hiểu thì thấy update nó validate , và sau khi bị anh Trung hành, tìm hiểu alias của update em mới thấy lúc đầu là do em viết thiếu chữ "s" nên nó mới thành method not valid